syncer/syncer.go

```diff
@@ -235,22 +195,13 @@ func NewSyncer(cfg *config.SubTaskConfig) *Syncer {
 	syncer.tracer = tracing.GetTracer()
 	syncer.setTimezone()
 	syncer.addJobFunc = syncer.addJob
+	syncer.enableRelay = enableRelay
 
 	syncer.checkpoint = NewRemoteCheckPoint(syncer.tctx, cfg, syncer.checkpointID())
 
-	syncer.syncCfg = replication.BinlogSyncerConfig{
-		ServerID:                uint32(syncer.cfg.ServerID),
-		Flavor:                  syncer.cfg.Flavor,
-		Host:                    syncer.cfg.From.Host,
-		Port:                    uint16(syncer.cfg.From.Port),
-		User:                    syncer.cfg.From.User,
-		Password:                syncer.cfg.From.Password,
-		UseDecimal:              true,
-		VerifyChecksum:          true,
-		TimestampStringLocation: syncer.timezone,
-	}
+	syncer.setSyncCfg()
```

Comment:
Is `shardingSyncCfg` not needed anymore?

Reply:
yes, I remove it in 338f680